.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+jobs:
+  build: # runs not using Workflows must have a `build` job as entry point
+    docker: # run the steps with Docker
+      - image: circleci/openjdk:10.0.2-13
```

Comment:
Would we want to run both openjdk8 and 10? or is that out of scope?

Reply:
Let's start with 10. Second-best case scenario, Travis-CI stays on until we're ready to switch everything to 10 and we'll never need to think about 8 on Circle